app/views/comments/_comment_actions.html.erb

```diff
@@ -17,7 +17,7 @@
       <%= edit_comment_link comment %>
     </li>
   <% end %>
-  <% if comment.pseud.nil? and (logged_in_as_admin? || is_author_of?(comment.ultimate_parent)) %>
+  <% if comment.pseud.nil? and ((logged_in_as_admin? && policy(comment).can_delete_comment?) || is_author_of?(comment.ultimate_parent)) %>
```

Comment:
Should this be `policy(comment).can_mark_comment_spam?` They're functionally the same at the moment, but it seems like it would be better to use the proper one in case they ever deviate.

Reply:
Yeah, good call. Thanks for the catch